modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractPhpCodegen.java

```diff
@@ -285,6 +286,11 @@ public void processOpts() {
 
         // all PHP codegens requires Composer, it means that we need to exclude from SVN at least vendor folder
         supportingFiles.add(new SupportingFile("gitignore", "", ".gitignore"));
+        // Add declareStrictTypes property to the additionalProperties
+    if (additionalProperties.containsKey("declareStrictTypes")) {
+        this.declareStrictTypes = Boolean.parseBoolean(additionalProperties.get("declareStrictTypes").toString());
+    }
+    additionalProperties.put("declareStrictTypes", declareStrictTypes);
```

Comment:
what about adding the option in php client codegen instead?

https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/PhpClientCodegen.java#L86